pkgs/by-name/as/asfa/package.nix

```
@@ -0,0 +1,59 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkgs,
+  ...
```

Packages inside nixpkgs are usually imported using `pkgs.callPackage package.nix {}` which populates this top attrset with the values of `pkgs`, e.g. `rustPlatform` or `fetchFromGitHub`. This means you can also use this for the dependencies.

```suggestion
  gawk,
  help2man,
  openssl,
  libssh2,
```

---

Really appreciate the detailed feedback, learning a lot! Adapted.